package.json

```diff
@@ -23,7 +23,6 @@
   ],
   "scripts": {
     "start": "(cd examples/main && (path-exists node_modules || npm i) && npm run start-local)",
-    "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5 dist/esm dist/es6",
```

Comment:
Probably for another PR, but don't we want a `scripts/clean.sh?` or `scripts/build.sh clean`?

Reply:
No reason to keep it separate. We published the last alpha with redundant files because `clean` was not called with `build`.

Reply:
No reason to keep it separate. 

> Not for build purposes. My thinking was that a typical usage for a `clean` script is to let the user to clean out built and installed artifacts from repo / the source tree, as they can take a lot of space. Not critical.